packages/sdk/src/types/attributes/ConnectorAttribute.ts

```diff
@@ -6,6 +6,7 @@ export interface ConnectorAttribute {
     succeededBy?: string;
     deletionInfo?: ConnectorAttributeDeletionInfo;
     shareInfo?: ConnectorAttributeShareInfo;
+    isDefault?: boolean;
```

Comment:
I think this property should also be added to some Requests, for example, to the `GetOwnRepositoryAttributeRequest` or the `GetAttributesRequest`. In the `GetAttributesRequest`, there also seem to miss other properties.

Reply:
IMO we should have a flag to stop the runtime automatically assigning defaults to attributes and then use this flag in the connector. In this case we can remove this everywhere in the connector.

Reply:
I close this as we don't want to expose more of the `isDefault` feature set to the connector.